vagrant/provisioning/arkcase-ee-foia-AWS.yml

```diff
@@ -31,10 +31,12 @@
       tags: [core, solr]
     - role: pentaho-setup
       tags: [core, pentaho, pentaho-setup]
-    - role: pentaho-ce
-      tags: [core, pentaho, pentaho-ce]
+    - role: pentaho-ee
+      tags: [core, pentaho, pentaho-ee]
     - role: pentaho-configuration
       tags: [core, pentaho, pentaho-configuration]
+    - role: pentaho-license
+      tags: [core, pentaho, pentaho-license]
```

Comment:
Do you also include the pentaho_ee_license_year fact in the facts file?
If not then the licenses for Pentaho won't be installed

Reply:
@dzdimov Yes, there is a `pentaho_ee_license_year: 2020` in the facts file.